ios/Runner/Info.plist

```diff
@@ -41,5 +41,23 @@
 	</array>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<false/>
+
+	<key>CFBundleURLTypes</key>
+    <array>
+        <dict>
+            <key>CFBundleTypeRole</key>
+            <string>Editor</string>
+            <key>CFBundleURLName</key>
+            <string>tech.pylons.easel</string>
+            <key>CFBundleURLSchemes</key>
+            <array>
+                <string>pylons-pylons_easel</string>
```

Comment:
Only write easel. Ios app will not work if the schema contains _

Reply:
I'll suggest that this should be stated in the SDK documentation

Reply:
Yes you are right and i will add it today. Thanks for mentioning